sources/packages/api/src/services/institution-location/institution-location.models.ts

```diff
@@ -11,12 +11,22 @@ export interface LocationWithDesignationStatus {
 }
 
 /**
- * Service model for institution location.
+ * Service model for institution location for AEST and create.
  */
 export interface InstitutionLocationModel extends AddressInfo {
   locationName: string;
   institutionCode: string;
-  primaryContactFirstName: string;
+  primaryContactFirstName?: string;
+  primaryContactLastName: string;
```

Comment:
Is it possible for primaryContactFirstName to be optional while primaryContactLastName and primaryContactEmail are not?

Reply:
you are right. it should not be optional.